src/constants/MarkdownFiles/posts/2025-09-12-gsoc-25-bishoy-wadea-final-report.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "GSoC’25 Final Report by Bishoy Wadea"
+excerpt: "Developing 10 Math Sugar Activities"
+category: "DEVELOPER NEWS"
+date: "2025-09-12"
+slug: "2025-09-12-gsoc-25-bishoy-wadea-final-report"
+author: "@/constants/MarkdownFiles/authors/bishoy-wadea.md"
+tags: "gsoc25,sugarlabs,Final-Report,BishoyWadea"
+image: "assets/Images/GSOC.webp"
+---
+
+# Google Summer of Code 2025: Educational Activities for Sugar
+
+Name - Bishoy Wadea
```

Comment:
This styling is inconsistent with all the other blogs.

In your previous blog, you have:

```
**Project:** [AI Organizer](https://github.com/Bishoywadea/AI-Organizer)  
**Mentors:** [Ibiam Chihurumnaya](https://github.com/chimosky)  
**Assisting Mentor:** [Walter Bender](https://github.com/walterbender/)  
**Reporting Period:** 2025-09-05 – 2025-09-11 
```